app/services/file_appender.rb

```diff
@@ -125,6 +126,11 @@ def create_node(file)
       node
     rescue StandardError => error
       message = "#{self.class}: Failed to append the new file #{original_filename} for #{node.id} to resource #{parent.id}: #{error}"
+      # For FileSet parents we're likely attaching a derivative, so let the
+      # error fall through so sidekiq restarts.
+      raise FileUploadFailed, message if file_set?(parent)
+      # Prevent error fallthrough for resource file attachments - otherwise if
+      # this keeps retrying we'll end up with a bunch of orphan files on disk.
```

Comment:
Suggestion: Add a little more here about the use case that results in the orphan files. Specifically it's because of the files that were ingested in the loop before we hit this one that's failing.

Suggestion: Use "do not raise" instead of "prevent error fallthrough"

Reply:
Done - take a look?